body/index.ts

```diff
@@ -0,0 +1,102 @@
+import { Readable } from "stream";
+
+export type GetBodyOpts = {
+  encoding: string;
+  limit?: number;
```

Comment:
Required?

Reply:
Not exactly, but I think it feels strange to control the limit inside of body